libvast/builtins/formats/json.cpp

```diff
@@ -640,8 +645,13 @@ class selector_parser final : public virtual parser_parser_plugin {
 
   auto parse_parser(parser_interface& p) const
     -> std::unique_ptr<plugin_parser> override {
-    argument_parser{name()}.parse(p);
+    auto parser
+      = argument_parser{name(), fmt::format("https://vast.io/docs/next/"
+                                            "formats/{}",
```

Comment:
This link must be adapted to https://docs.tenzir.com/next/formats/json

Reply:
Ok. i Have also changed other occurrences in the code as this was no effort just replacing them

Reply:
Thanks, that's fantastic!